### ISSUE TITLE
ReadOnlySpan<T>.StartsWith() example show wrong result

### ISSUE THREAD

**Issue by RickKra** [dotnet/docs#43024]
### Type of issue

Typo

### Description

That page says this:

ReadOnlySpan<char> text = "some arbitrary text";
return text.StartsWith('"') && text.EndsWith('"'); // false

Shouldn't the comment indicate that the result is true?

### Page URL

https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-9/libraries

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/core/whats-new/dotnet-9/libraries.md

### Document Version Independent Id

0c8c7059-86a1-e5df-32e6-fae6fdc5cae0

### Article author

@gewarren

### Metadata

* ID: 7c184200-3411-aa5b-ba3f-0b4a3ce0f589 
* Service: **dotnet-fundamentals**

**Comment by Marusyk** [comment]
@RickKra  why do you think it should be `true`?

**Comment by RickKra** [comment]
Sorry, I spaced that the quotes are not stored with the string. My apologies for wasting your time on this.

**Comment by Marusyk** [comment]
no problem, does it make sense to close the issue?

**Comment by RickKra** [comment]
Yes it does. Is that something I'm supposed to do or will you handle it?
